user/pages/03.commerce2/02.developer-guide/09.promotions/02.create-a-condition/docs.md

```diff
@@ -4,4 +4,222 @@ taxonomy:
     category: docs
 ---
 
+Commerce conditions are used in when creating a Promotion or a Payment Gateway 
+for your store. 
+
+In this context, conditions are a set of plugins unique to the 
+commerce project. They are not connected in function or code to conditions in 
+core such as: Drupal\Core\Database\Query\Condition. 
```

Comment:
I like the idea of clarifying what they are but it struck me as odd to see them compared to database conditions. What if we spoke more about what they *are*, like the use cases for conditions that exist already? Ie, we could say that they're used to determine applicable promotions, payment gateways, and shipping methods for orders. For example, conditions may limit a promotion by the order total or types of products on the order.